OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -506,6 +564,11 @@ public bool CanEnterCell(CPos cell, Actor ignoreActor = null, bool checkTransien
 			return Info.CanEnterCell(self.World, self, cell, ignoreActor, checkTransientActors ? CellConditions.All : CellConditions.BlockedByMovers);
 		}
 
+		public bool CanMoveFreely(CPos cell, Actor ignoreActor = null, bool checkTransientActors = true)
```

Comment:
This method name doesn't sit well with me.  With `CanEnterCell` it is clear what it does and how the arguments relate to it.  `CanMoveFreely` is a bit ambiguous.